test/unit/core/network/createNetwork.spec.js

```diff
@@ -18,58 +18,44 @@ describe("createNetwork", () => {
     propertyID: "mypropertyid"
   };
 
-  const logger = console.log;
+  const logger = console;
 
   const nullLifecycle = {
     onBeforeSend: () => Promise.resolve(),
-    onResponseFragment: () => Promise.resolve()
+    onResponse: () => Promise.resolve()
   };
 
   it("calls interact by default", done => {
     const networkStrategy = url => {
-      return new Promise(() => {
-        expect(url).toEqual(
-          "https://alloy.mysite.com/interact?propertyID=mypropertyid"
-        );
-        done();
-      });
+      expect(url).toEqual(
+        "https://alloy.mysite.com/interact?propertyID=mypropertyid"
+      );
+      done();
+      return Promise.resolve();
     };
-    const network = createNetwork(
-      config,
-      logger,
-      nullLifecycle,
-      networkStrategy
-    );
-    const { send } = network.newRequest();
-    send();
```

Comment:
Some of the changes to the file, like the changes to this test, were for code simplification.